optimizing/index.md

```diff
@@ -53,39 +53,42 @@ using BenchmarkTools
+The Chairmarks.jl package provides convenient syntax to do just that.
 
-### BenchmarkTools
+### Chairmarks
 
-[BenchmarkTools.jl](https://github.com/JuliaCI/BenchmarkTools.jl) is the most popular package for repeated measurements on function executions.
-Similarly to `@time`, BenchmarkTools offers `@btime` which can be used in exactly the same way but will run the code multiple times and provide an average.
-Additionally, by using `$` to [interpolate external values](https://juliaci.github.io/BenchmarkTools.jl/stable/manual/#Interpolating-values-into-benchmark-expressions), you remove the overhead caused by global variables.
+[Chairmarks.jl](https://github.com/LilithHafner/Chairmarks.jl) is the latest and greatest benchmarking suite used to make fast and accurate timing measurements.
+Chairmarks offers `@b` (for "benchmark") which can be used in the same way as `@time` but will run the code multiple times and provide a minimum execution time.
+Alternatively, Chairmarks also provides `@be` to run the benchmark and output all of its statistics.
 
-```>$-example
-using BenchmarkTools
-@btime sum_abs(v);
-@btime sum_abs($v);
+```>chairmarks-example
+using Chairmarks
+@b sum_abs(v)
+@be sum_abs(v)
+```
+
+Chairmarks supports a pipeline syntax with optional `init`, `setup`, `teardown`, and `keywords` arguments for more extensive control over the benchmarking process.
+The `sum_abs` function could also be benchmarked using pipeline syntax as below.
+
+```>pipeline-example-simple
+@be v sum_abs
 ```
 
-In more complex settings, you might need to construct variables in a [setup phase](https://juliaci.github.io/BenchmarkTools.jl/stable/manual/#Setup-and-teardown-phases) that is run before each sample.
-This can be useful to generate a new random input every time, instead of always using the same input.
+For a more complicated example, you could write the following to benchmark a matrix multiplication function for one second, excluding the time spent to *setup* the arrays.
 
-```>setup-example
+```>pipeline-example-complex
 my_matmul(A, b) = A * b;
-@btime my_matmul(A, b) setup=(
-    A = rand(1000, 1000); # use semi-colons between setup lines
-    b = rand(1000)
-);
+@be (A=rand(1000,1000), b=rand(1000)) my_matmul(_.A, _.b) seconds=1
 ```
 
-For better visualization, the `@benchmark` macro shows performance histograms:
+See the [Chairmarks documentation](https://chairmarks.lilithhafner.com/) for more details on benchmarking options.
+For better visualization, [PrettyChairmarks.jl](https://github.com/astrozot/PrettyChairmarks.jl) shows performance histograms alongside the numerical results.
 
 \advanced{
 Certain computations may be [optimized away by the compiler]((https://juliaci.github.io/BenchmarkTools.jl/stable/manual/#Understanding-compiler-optimizations)) before the benchmark takes place.
 If you observe suspiciously fast performance, especially below the nanosecond scale, this is very likely to have happened.
 }
```

Comment:
This can now be removed (or updated for Chairmarks @LilithHafner ?).

Reply:
Maybe just clarify that the explanation from BenchmarkTools applies here too